product-service/services/products.js

```diff
@@ -1,12 +1,72 @@
-const productData = require('./products.json');
+import { client } from '../db/client';
 
 class ProductsService {
     async getList() {
-        return Promise.resolve(productData);
+        try {
+            await client.connect();
+                   
+            const products = await client.query(`
+                SELECT * FROM products p
+                INNER JOIN stocks s
+                ON s.product_id = p.id
+            `);
+
+            return Promise.resolve(products.rows);
+        } catch (e) {
+            console.log(e);
+
+            throw e;
+        }
     }
 
     async getOne(productId) {
-        return Promise.resolve(productData.find(({id}) => id === productId));
+        try {
+            await client.connect();
+                   
+            const products = await client.query(`
+                SELECT * FROM products p
+                INNER JOIN stocks s
+                ON s.product_id = p.id
+                WHERE p.id = $1::uuid
+            `, [productId]);
+
+            return Promise.resolve(products.rows[0]);
+        } catch (e) {
+            console.log(e);
+
+            throw e;
+        }
+    }
+
+    async create({ count, price, title, description }) {
```

Comment:
I see errors in Postman when I call this [method](https://elbwtkyhc4.execute-api.eu-west-1.amazonaws.com/dev/products). 
![image](https://user-images.githubusercontent.com/4805000/181434695-3336ef11-1ce2-4b8b-9169-92e0d0a57c9a.png)

Though validation of body in the request is correct.

Reply:
It's because PR is still in draft and work is in progress. Thank you for the patience